Fix isw raid0 incorrect  sectors calculation, thanks to Valentin Pavlyuchenko
--- 1.0.0.rc16-3/dmraid/lib/format/ataraid/isw.c
+++ 1.0.0.rc16-3/dmraid/lib/format/ataraid/isw.c
@@ -776,7 +776,9 @@ _create_rd(struct lib_context *lc,
 	r->di = rd->di;
 	r->fmt = rd->fmt;
 	r->offset = dev->vol.map[0].pba_of_lba0;
-	if ((r->sectors = dev->vol.map[0].blocks_per_member - RAID_DS_JOURNAL))
+	//fix bugs on ICH10R
+	//if ((r->sectors = dev->vol.map[0].blocks_per_member - RAID_DS_JOURNAL))
+	if ((r->sectors = dev->vol.map[0].blocks_per_member))
 		goto out;
 
 	log_zero_sectors(lc, rd->di->path, handler);
